friends/friendModel.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+const definition = {
+    firstName: {
+        type: String,
+        required: true
+    },
+    lastName: {
+        type: String,
+        required: true 
+    },
+    age: {
```

Comment:
Consider using a custom validator for the age:
Reference this documentation: http://mongoosejs.com/docs/validation.html#custom-validators